### PR TITLE
added new folders

### PR DESCRIPTION
**Pull request by gaaarfild**

### CODE REVIEW

File: Laravel.gitignore [hunk 1]
```diff
@@ -2,3 +2,13 @@
 .env.*.php
 .env.php
 .env
+/node_modules
+/.idea/
```
Comment (shiftkey) [inline Laravel.gitignore]: We've bumped up against some rule updates to the JetBrains IDEs - see #1475 for the details.

File: Laravel.gitignore [hunk 2]
```diff
@@ -2,3 +2,13 @@
...
+/.rocketeer/
+/.phpintel/
+/vendor/compiled.php
+/vendor/services.json
+/bootstrap/cache/
+!bootstrap/cache/.gitkeep
+.directory
+Thumbs.db
```
Comment (shiftkey) [inline Laravel.gitignore]: We're tracking the rules for OS X under `Global/OSX.gitignore`

**Comment by shiftkey** [comment]
@gaaarfild apologies for not getting back to this sooner, and thanks for the effort. There's a merge conflict here which will prevent merging once this is ready to merge - are you interested in resolving those conflicts?

Also, as I'm not familiar with the ins and outs of Laravel, I'm wondering if it's worth annotating these rules with comments, to help others understand what the rules are there for. Do you think that's worth doing here?

**Comment by shiftkey** [comment]
I'm trying to clear out the backlog of PRs here, and given the comments haven't seen a reply I'm going to close this out.

@gaaarfild please let me know if you're interested in working together so this can get merged in.

**Comment by gaaarfild** [comment]
@shiftkey Hi! Sorry for late answer. Was incredibly busy. 
I can create one more PR wit updated branch to avoid conflicts,

**Comment by shiftkey** [comment]
@gaaarfild that'd be great - thanks! :heart:

**Pull request closed without merging**